dts/bindings/i3c/nxp,mcux-i3c.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2019 NXP
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP MCUX I3C controller
+
+compatible: "nxp,mcux-i3c"
+
+include: [i3c-controller.yaml, pinctrl-device.yaml]
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    i3c-od-scl-hz:
+      type: int
+      required: false
+      description: |
+        Open Drain Frequency for the I3C controller. When undefined, use
+        the controller default or as specified by the I3C specification.
+
+    i3c-scl-hz:
```

Comment:
why is this here, doesn't this come from `i3c-controller.yaml`?

Reply:
This is supposed to be set to be a required property, and yet I forgot (after copy-and-paste). Will change.

Reply:
Removed.